src/styles/styles.css

```diff
@@ -3,6 +3,35 @@
 
 }
 
+#app .container {
+ width: 99%;
+}
+
+#app .container .row {
+padding: 40px 40px;
+}
+
+.header {
+    color: black;
+    background-color: gainsboro;
+}
+
+.logo {
+   content:url(https://www.tikalk.com/assets/logo-1bb9a466c9d195c148557434c3437446.png);
```

Comment:
better to use a background image .. An image inside a content attribute can't be manipulated.  and we should definitely change the logo to the one with the octopus.